mautic/mautic
# Added report column: Date added to company

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR added new column in reports. Along  with company fields added **Date added to company** column

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create contacts report with Company Name and Date added to company column
3. Assign companie sto few contacts 
4. Check report If works properly
